tests/lib/rules/jsx-curly-brace-presence.js

```diff
@@ -241,6 +241,26 @@ ruleTester.run('jsx-curly-brace-presence', rule, {
     {
       code: ['<a a={"start\\', '\\', 'end"}/>'].join('/n'),
       options: ['never']
+    },
+    {
+      code: `
+        <App prop={\`
+          a
+          b
+        \`} />
+      `,
+      options: ['never']
```

Comment:
can we add similar test cases for the "always" option?

Reply:
Added